sdk/storage/azure-storage-blobs/NOTICE.txt

```diff
@@ -269,62 +234,267 @@ License notice for
+Copyright (c) 2013 - 2019, Linus Nielsen Feltzing, <linus@haxx.se>
+Copyright (c) 1998 - 2016, Vijay Panghal, <vpanghal@maginatics.com>
+Copyright (c) 2009, Markus Moeller, <markus_moeller@compuserve.com>
+Copyright (c) 1998 - 2019, Florin Petriuc, <petriuc.florin@gmail.com>
+Copyright (c) 2014, Bill Nagel <wnagel@tycoint.com> , Exacq Technologies
+Copyright (c) 2009, 2011, Markus Moeller, <markus_moeller@compuserve.com>
+Copyright symbol VALUE License', https://curl.haxx.se/docs/copyright.html
+Copyright (c) 1998, 1999, 2017 Kungliga Tekniska Hogskolan (Royal Institute of Technology, Stockholm, Sweden).
+Copyright (c) 1995, 1996, 1997, 1998, 1999 Kungliga Tekniska Hogskolan (Royal Institute of Technology, Stockholm, Sweden).
 
-Copyright (c) 2015 Max Woolf
+COPYRIGHT AND PERMISSION NOTICE
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+Copyright (c) 1996 - 2020, Daniel Stenberg, <daniel@haxx.se>, and many
+contributors, see the THANKS file.
 
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
+All rights reserved.
+
+Permission to use, copy, modify, and distribute this software for any purpose
+with or without fee is hereby granted, provided that the above copyright
+notice and this permission notice appear in all copies.
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT OF THIRD PARTY RIGHTS. IN
+NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
+OR OTHER DEALINGS IN THE SOFTWARE.
+
+Except as contained in this notice, the name of a copyright holder shall not
+be used in advertising or otherwise to promote the sale, use or other dealings
+in this Software without prior written authorization of the copyright holder.
+
+
+---------------------------------------------------------
+
+---------------------------------------------------------
+
+azure/azure-c-shared-utility 8ee0e5d88d1771d33e36d38d392ec9d9ae13ea55 - MIT
```

Comment:
"Azure/..." and "Microsoft/..." packages are not considered 3rd party OSS.  We do not need to list them.  Did you see or receive direction from CELA stating otherwise?

Reply:
Sure, I'll remove `azure/azure-c-shared-utility`, `azure/macro-utils-c`, `azure/umock-c` and `microsoft/wil`.